Iguana2024/devops-2024
# Igd 18 set up scheduler

## What type of PR is this? (check all applicable)


## Description
Added:
"(sudo -u vagrant crontab -l 2>/dev/null; echo "* * * * * /bin/bash /vagrant/crontab.sh > /vagrant/output.log 2>&1") | sudo -u vagrant crontab -"
"(sudo -u vagrant crontab -l 2>/dev/null; echo "*/5 * * * * /usr/bin/python3 /vagrant/report.py") | sudo -u vagrant crontab -"
on lines 44 and 45 in Vagrantfile

## QA Instructions, Screenshots, Recordings

To ensure that everything is working, use commands " ssh -i path_to_your_private_key vagrant@ip_of_machine" 
![image](https://github.com/Iguana2024/devops-2024/assets/64985383/8267a8f8-8482-4024-a3fb-53b31c8b0b5e)
write the command  "crontab -l"
![image](https://github.com/Iguana2024/devops-2024/assets/64985383/2ebf1a02-375e-4a78-8bf7-399b9c4d88ed)
if the output matches - then everything is fine

## [optional] Are there any post deployment tasks we need to perform?
